uber/decorators.py

```diff
@@ -234,6 +233,10 @@ def with_restrictions(*args, **kwargs):
                 if not cherrypy.session.get('staffer_id'):
                     raise HTTPRedirect('../signups/login?message=You+are+not+logged+in')
 
```

Comment:
I'm writing up a fix for this now, I'll test and push back to this branch and then merge.